test/mpi/autogen.sh

```diff
@@ -62,6 +62,14 @@ check_copy() {
     fi
 }
 
+create_benchmarks() {
+    MYDEF_BOOT=$PWD/../../modules/mydef_boot
+    export PATH=$MYDEF_BOOT/bin:$PATH
+    export PERL5LIB=$MYDEF_BOOT/lib/perl5
+    export MYDEFLIB=$MYDEF_BOOT/lib/MyDef
+    (cd bench && ./autogen.sh)
+}
```

Comment:
The concern is that the user may only download the testsuite tarball and this make the autogen broken. How about protecting it with -
```
if test -d `$MYDEF_BOOT` ; then
        export PATH=$MYDEF_BOOT/bin:$PATH
        export PERL5LIB=$MYDEF_BOOT/lib/perl5
        export MYDEFLIB=$MYDEF_BOOT/lib/MyDef
fi
```
MyDef is essentially a dependency. User (like me) always can install MyDef in his system then `autogen.sh` will always work.

Reply:
Done.